apps/services/user-profile/src/main.ts

```diff
@@ -1,19 +1,11 @@
 import { bootstrap } from '@island.is/infra-nest-server'
 import { AppModule } from './app/app.module'
 import { openApi } from './openApi'
-import * as Sentry from '@sentry/node'
 import { environment } from './environments'
-import { SentryInterceptor } from '@island.is/infra-monitoring'
-
-Sentry.init({
-  dsn: environment.sentry.dsn,
```

Comment:
Get rid of this environment field and infra secret as well?

Reply:
fixed